sources/osg/StateAttribute.js

```diff
@@ -33,7 +37,9 @@ define( [
             return this.attributeType;
         },
 
-        apply: function () {},
+        apply: function () {
+            this.dirty( false );
```

Comment:
I don't think I like that... we compute a code from a string (by iterating on a string) each time we apply an attribute.

We apply way more attribute than we generate a shader I think...

+ we have to make sure all the other apply will call the original stateAttribute apply (error prone?)